posthog/api/team.py

```diff
@@ -196,6 +201,13 @@ def get_has_group_types(self, team: Team) -> bool:
     def get_groups_on_events_querying_enabled(self, team: Team) -> bool:
         return groups_on_events_querying_enabled()
 
+    def get_jwt_token(self, team: Team) -> Optional[str]:
+        return encode_jwt(
+            {"team_id": team.id},
+            timedelta(days=7),
```

Comment:
Also probably good to have this as a setting but not the end of the world

Reply:
Ah good call, can do.